### DISCUSSION TITLE
Blobs in Octoprint

### DISCUSSION THREAD

**Discussion by Krisvl10** [Jyers/Marlin#759] [General Discussion]
![thumbnail_IMG_8733](https://user-images.githubusercontent.com/83925890/117587983-4ca85900-b0e6-11eb-8571-64df233106d0.jpg)

hi Jyer thanks for your great work with the firmware. I installed the firmware 4.2.2 5 * 5 manual mesh and we configured everything according to the wiki for octoprint as well as the mesh configuration and in the circular prints many blobs began to come out, the printer stopped for a few seconds and the material kept coming out that makes blobs appear. I printed from the SD card and thus there were no failures in the prints. Look for that in terminal if errors appeared and I found this:

first "Communication timeout while printing, trying to trigger printer response. Configure long-running commands or increase communication timeout if that occurs regularly on specific commands or long movements" and second "Octoprint too many lines to cushion the printing of cut blobs "

Try all firmware 4.2.2 5x5, 4.2.2 3x3 and 4.2.2 Default all with the same Blobs result on all prints. Finally I had to install the Creality firmware again and the prints stopped having these faults so I see that it is the jyers firmware that may not be communicating well with my printer.

What can I do to fix this problem? Do you have very good options that I would like to try? I'm sorry for my English, I'm from Colombia

**Comment by Jyers** [comment]
What plugin's do you have installed? Also have you tried increasing your timeouts as the error indicates?

**Comment by Krisvl10** [comment]
Hello Jyers, Thank you very much for answering. These are the plugins that I have currently installed in my Octoprint

Creality Temperature: 1.2.4 
Creality-2x-temperature-reporting-fix: 0.0.4 
Dashboard Plugin: 1.18.2 
DetailedProgress Plugin: 0.2.7 
DisplayLayerProgress Plugin: 1.26.0 
File Check Plugin: 2021.2.23 
Firmware Check Plugin: 2021.2.4 
M73 Progress Plugin: 0.2.1 
Octolapse: 0.4.1 _(Not used for firmware testing)_
Pi Support Plugin: 2021.3.26.post2 
and the M420 S1 command in the gcode, in addition to the octoprint settings commands

Regarding the waiting time, the truth is that I did not understand exactly where I should expand or configure it. I found a video that talks about this communication failure in octoprint directly in the advanced settings in the firmware, but I could not compile it to test (inexperience I attach the image). I also tried other settings directly on the cutter (cure) on mesh corrections like maximum resolution, offset resolution and maximum offset which makes the process more agile on complex process offsets but it didn't work. I appreciate any advice you can give me, thank you again.

![123@3x-100](https://user-images.githubusercontent.com/83925890/117902983-4bb52a00-b294-11eb-90b9-f607ca28059f.jpg)

**Reply by Jyers** [reply]
Well you can definitely remove all your creality fix type plugins as those are not needed for this firmware. Maybe that will solve your connection issue too.

**Reply by Krisvl10** [reply]
Hello Jyers, deactivate the plugins and effectively the impressions improved, I tested the firmware 4.2.2 default and 4.2.2 3x3 and did a test with Octolapse to see how it occurred and the reduction of spots and its decrease and improvement was notable, in addition to the pauses In the impressions by layer change I did not perceive them however in the impression of 4.2.2 3x3 very very small traces came out and I will continue testing the same as in octolapse. Thank you for your support

![thumbnail_IMG_8776](https://user-images.githubusercontent.com/83925890/118204645-530b3d80-b424-11eb-8753-f0d396561090.jpg)

**Comment by Tuncay-Ayhan** [comment]
> ![thumbnail_IMG_8733](https://user-images.githubusercontent.com/83925890/117587983-4ca85900-b0e6-11eb-8571-64df233106d0.jpg)
> 
> hi Jyer thanks for your great work with the firmware. I installed the firmware 4.2.2 5 * 5 manual mesh and we configured everything according to the wiki for octoprint as well as the mesh configuration and in the circular prints many blobs began to come out, the printer stopped for a few seconds and the material kept coming out that makes blobs appear. I printed from the SD card and thus there were no failures in the prints. Look for that in terminal if errors appeared and I found this:
> 
> first "Communication timeout while printing, trying to trigger printer response. Configure long-running commands or increase communication timeout if that occurs regularly on specific commands or long movements" and second "Octoprint too many lines to cushion the printing of cut blobs "
> 
> Try all firmware 4.2.2 5x5, 4.2.2 3x3 and 4.2.2 Default all with the same Blobs result on all prints. Finally I had to install the Creality firmware again and the prints stopped having these faults so I see that it is the jyers firmware that may not be communicating well with my printer.
> 
> What can I do to fix this problem? Do you have very good options that I would like to try? I'm sorry for my English, I'm from Colombia

This is due to the **"Detailed Progress plugin"**. Had the same issue. Turned it off and it was gone. It prints normal, but sometimes it buffers and stops while printing, leaving blobs around your print.

What kind of Raspberry Pi do you have? I think it's a limitation of the Pi 3B+ . Maybe someone else can also confirm this while they didn't knew that this was the case.

**Reply by Krisvl10** [reply]
Hello Friend thanks for your answer, I also use raspi 3b + and I also deactivated your suggestion and effectively improved the prints with the firmware. I will continue testing with octolapse to see how communication works with this active process.

**Reply by Tuncay-Ayhan** [reply]
> Hello Friend thanks for your answer, I also use raspi 3b + and I also deactivated your suggestion and effectively improved the prints with the firmware. I will continue testing with octolapse to see how communication works with this active process.

Great!
